webdriver/tests/get_element_shadow_root/META.yml

```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1168659
```

Comment:
Linking directly to https://bugs.chromium.org/p/chromedriver/issues/detail?id=3715 might be better.